R/uvhydrograph-render.R

```diff
@@ -22,7 +22,9 @@ uvhydrographPlot <- function(reportObject) {
     for (month in months) {
       primary <- getPrimaryReportElements(reportObject, month, timezone, excludeZeroNegativeFlag)
       
-      if(!is.null(primary[['plot']])){
+      
+      
+      if(!is.null(primary[['plot']]) && useSecondaryPlot(reportObject)){
```

Comment:
I've been in the habit of replacing pretty much all `is.null()` calls with `isEmptyOrBlank` calls, though if this should only happen if it is null and not if it is just empty then it should be kept as is.